docs/index.md

```diff
@@ -69,33 +75,31 @@ If necessary, provision all OSG hosts that are in your site plan and that do not
     -   [Submitting jobs to HTCondor-CE](compute-element/submit-htcondor-ce)
 -   [`osg-configure` Reference](other/configuration-with-osg-configure)
 
-### Installing and Configuring Other Nodes ###
+### Installing and Configuring Other Services ###
 
-All of these node types and their services are optional, although OSG requires the Frontier Squid caching service if you have installed [CVMFS](worker-node/install-cvmfs) on your worker nodes.
+All of these node types and their services are optional, although OSG requires a HTTP caching service if you have
+installed [CVMFS](worker-node/install-cvmfs) on your worker nodes.
 
--   [Install Frontier Squid, the HTTP caching proxy service](data/frontier-squid)
--   RSV monitoring to monitor and report to OSG on the health of your site
-    -   [Install RSV](monitoring/install-rsv)
--   [Install the GlideinWMS VO Frontend](other/install-gwms-frontend) if your want your users’ jobs to run on the OSG
-    -   [Install the RSV GlideinWMS Tester](monitoring/install-rsv-gwms-tester) if you want to test your front-end's ability to submit jobs to sites in the OSG
--   Storage element (pick one):
-    -   GridFTP
+-   [Install Frontier Squid](data/frontier-squid), a HTTP caching proxy service.
+-   Storage element:
+    -   Existing POSIX-based systems (such as NFS, Lustre, or GPFS):
         -   [Install standalone OSG GridFTP](data/gridftp): GridFTP server
         -   (optional) [Install load-balanced OSG GridFTP](data/load-balanced-gridftp): when a single GridFTP server isn't enough
-    -   BeStMan
-        -   [Install Bestman on POSIX](data/bestman-install): BeStMan2 SRM server + GridFTP server
-        -   [Install Bestman on Hadoop](data/install-hadoop-2-0-0): BeStMan2 SRM and GridFTP servers on the Hadoop Distributed File System
-    -   Hadoop Distributed File System (HDFS)
+    -   Hadoop Distributed File System (HDFS):
         -   [Hadoop Overview](data/hadoop-overview): HDFS information, planning, and guides
-    -   XRootD
+    -   XRootD-based storage:
```

Comment:
I don't think this change is needed. Reader already knows they're looking at storage, and if you add the suffix here, you'd need to add the suffix to the other methods (e.g. HDFS-based storage) to remain consistent.

Reply:
Good point.

XRootD is always confusing because it is both a storage element and a data access technology that lives on top of other storage elements.

However, the top of the list says "storage element" explicitly, so I guess changes here are redundant.